Uplift/Views/Footers/GymCellFooter.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  GymCellFooter.swift
+//  Uplift
+//
+//  Created by Elvis Marcelo on 3/22/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class GymCellFooter: UIView {
+    private let hoursLabel = UILabel()
+    private let locationNameLabel = UILabel()
+    private let statusLabel = UILabel()
+    private let capacityStatusLabel = UILabel()
+    private let capacityCountLabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .white
+        setupViews()
+        setupConstraints()
+    }
+
+    func configure(for gym: Gym) {
+        locationNameLabel.text = gym.name
+
+        //set gym status
+        let changingSoon = gym.isStatusChangingSoon()
+        if gym.isOpen {
+            if changingSoon {
+                statusLabel.textColor = .accentOrange
+                statusLabel.text = ClientStrings.CommonStrings.open
+            } else {
+                statusLabel.textColor = .accentOpen
+                statusLabel.text = ClientStrings.CommonStrings.open
+            }
+        } else {
+            if changingSoon {
+                statusLabel.textColor = .accentOrange
+                statusLabel.text = ClientStrings.CommonStrings.closed
+            } else {
+                statusLabel.textColor = .accentClosed
+                statusLabel.text = ClientStrings.CommonStrings.closed
+            }
```

Comment:
We can tidy this up a bit by doing something a long the lines of 
`statusLabel.textColor = gym.changingSoon ? .accentOrange : accentOpen`
`statusLabel.text = gym.isOpen ? ClientStrings.CommonStrings.open : ClientStrings.CommonStrings.closed`